src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.tsx

```diff
@@ -307,13 +307,14 @@ export const VenueRoomsEditor: React.FC<VenueRoomsEditorProps> = ({
       {!backgroundImage ? (
         <MapBackgroundPlaceholder />
       ) : (
-        <img
-          alt="draggable background "
-          className={`Container__background-image ${backgroundImageClassName}`}
-          src={backgroundImage}
-        />
+        <div className="VenueRoomsEditor__background-image-wrapper">
+          <img
+            alt="draggable background "
+            className={`VenueRoomsEditor__background-image ${backgroundImageClassName}`}
```

Comment:
It's worth keeping className calculation in separate var

Reply:
fixed in 24f391d2ee6cf6ad1d3364c161e287325e75ab44